RNTester/RNTesterUnitTests/MockInstance.cpp

```diff
@@ -0,0 +1,104 @@
+//
+//  MockInstance.cpp
+//  RNTesterUnitTests
+//
+//  Created by Julio Cesar Rocha on 10/22/18.
```

Comment:
Can you use the standard copyright header used elsewhere in the repo?

```
// Copyright (c) Facebook, Inc. and its affiliates.
//
// This source code is licensed under the MIT license found in the
// LICENSE file in the root directory of this source tree.
```

Reply:
Done for all instances. Can you please approve? :)